src/azure-cli/azure/cli/command_modules/acr/connected_acr.py

```diff
@@ -0,0 +1,192 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from msrest.exceptions import ValidationError
+from knack.log import get_logger
+from knack.util import CLIError
+from azure.cli.core.commands.client_factory import get_subscription_id
+from ._utils import (
+    get_registry_by_name,
+    validate_managed_registry,
+    user_confirmation
+)
+
+DEFAULT_MODE = 'registry'
+DEFAULT_LOG_LEVEL = 'Info'
+DEFAULT_MESSAGE_TTL = 'P2D'
+
+logger = get_logger(__name__)
+
+
+def acr_connected_acr_create(cmd,
```

Comment:
Please add support for a list of client token resource ids. The user will provide the list of client token resource ids, the client will not be responsible for creating the tokens (unlike sync). This corresponds to `properties.clientTokenIds`.